Doc/library/stdtypes.rst

```diff
@@ -353,7 +353,7 @@ Notes:
    The numeric literals accepted include the digits ``0`` to ``9`` or any
    Unicode equivalent (code points with the ``Nd`` property).
 
-   See https://www.unicode.org/Public/15.0.0/ucd/extracted/DerivedNumericType.txt
+   See `DerivedNumericType-14.0.0.txt <https://www.unicode.org/Public/15.0.0/ucd/extracted/DerivedNumericType.txt>`_
```

Comment:
```suggestion
   See `the Unicode Standard <https://unicode.org/Public/UNIDATA/extracted/DerivedNumericType.txt>`_
```

* The current link text implies this is the name of the file, but it isn't, and also, after the merge commit, the title and link are now also out of sync with one another. Furthermore, as a reader I would assume with the current link text, it is referring to a file inside the Python distribution or repository, rather than an external standard. Instead, let's use a clearer, broader and more accurate and representative link title.
* Update the link to be perpetually up to date with the current version of the Unicode standard